enterprise/models.py

```diff
@@ -77,6 +77,35 @@ class Meta(object):
     history = HistoricalRecords()
     identity_provider = models.SlugField(null=True, blank=True, default=None)
```

Comment:
Same comment here, while I happen to be thinking of this: The `identity_provider` field of Enterprise Customer should reference a complete [`provider_id`](https://github.com/edx/edx-platform/blob/f806f3a/common/djangoapps/third_party_auth/models.py#L158-L162) such as "`saml-mycorp`" and not just a SAML `provider_slug` like "`mycorp`". That way, we can support non-SAML providers in the future if we need to (OAuth2, Active Directory, or other types). We should not tie ourselves unnecessarily to SAML if the code can be fully generic.

Right now, this field actually doesn't say what it references - it should have at least a comment explaining what sort of value is meant to be stored in this field, and that should be a `provider_id`.

If you want to consider this out of scope, that's fine, but it would be good to fix soon.